File: TaskManager/TaskManager_210042112/TaskManager_112/TaskManager_112/Issue.cs
using System;
using System.Collections.Generic;
using System.Globalization;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TaskManager_112
{
    public class Issue
    {
        public int ID { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public string Priority { get; set; }
        public string Status { get; set; }
        public DateTime CreationDate { get; set; }
        public DateTime LastModifiedDate { get; set; }
        public string AssignedTo { get; set; }
        public List<string> Tags { get; set; } //cause tags can be multiple
        public List<string> Comments { get; set; } //cause comments can be multiple
        public Issue(int id,string title, string description, List<string> tags)
        {
            ID = id;
            Title = title;
            Description = description;
            CreationDate = DateTime.Now;
            LastModifiedDate = DateTime.Now;
            Tags = tags.ToList();    
            Comments = new List<string>();

        }

    }
}

File: TaskManager/src/main/java/org.example/Issue.java
package org.example;

import java.util.ArrayList;
import java.util.Date;
import java.util.List;

public class Issue
{


    public String IssueID,Description,Title,Priority;
    public Date CreationDate;
    public Date LastModifiedDate;
    public String Tags[];
    public List<Comment> commentList =new ArrayList<>();
    public Status status;

    public Issue(String issueID, String description, String title, String priority, Date creationDate, Date lastModifiedDate, String[] tags, List<Comment> commentList, Status status) {
        IssueID = issueID;
        Description = description;
        Title = title;
        Priority = priority;
        CreationDate = creationDate;
        LastModifiedDate = lastModifiedDate;
        Tags = tags;
        this.commentList = commentList;
        this.status = status;
    }
}
